noggin/tests/unit/conftest.py

```diff
@@ -35,6 +35,20 @@ def ipa_cert():
         yield
 
 
+@pytest.fixture
+def add_hidden_group(logged_in_dummy_user):
+    hidden_group = app.config.get('HIDE_GROUPS_IN')
+    ipa = logged_in_dummy_user
+    # We need to use request here as our ipa.group_find method will filter the hidden groups out
```

Comment:
There's another way you can do that. You can allow the `group_find` caller to override `posix=True`. You can do something like:
```python
final_kwargs = {"posix": True, "not_in_group": []}
final_kwargs.update(kwargs)
final_kwargs["not_in_group"].append(current_app.config.get('HIDE_GROUPS_IN'))
return super().group_find(*args, **kwargs)
```
Then if you pass `posix=False` it will override the defaults. Sorry to make you amend this function again! :-D